buildNix.sh

```diff
@@ -30,22 +38,21 @@ dotnet sln OSPSuite.SimModel4Nix.sln remove src/OSPSuite.SysTool/OSPSuite.SysToo
 dotnet sln OSPSuite.SimModel4Nix.sln remove src/OSPSuite.XMLWrapper/OSPSuite.XMLWrapper.vcxproj
 dotnet sln OSPSuite.SimModel4Nix.sln remove tests/TestAppNetCore/TestAppNetCore.csproj
 
-cmake -BBuild/Release/x64/ -Hsrc/OSPSuite.SimModelNative/ -DCMAKE_BUILD_TYPE=Release -DFuncParserDir=Build/Release/x64/ -DEXT=$EXT -DCMAKE_OSX_ARCHITECTURES=x86_64
-cp packages/OSPSuite.FuncParser.$1/OSPSuite.FuncParserNative/bin/native/x64/Release/libOSPSuite.FuncParserNative.$EXT Build/Release/x64/
-cp packages/OSPSuite.SimModelSolver_CVODES.$1/OSPSuite.SimModelSolver_CVODES/bin/native/x64/Release/libOSPSuite.SimModelSolver_CVODES.$EXT Build/Release/x64/
-make -C Build/Release/x64/
-dotnet build OSPSuite.SimModel4Nix.sln /property:Configuration=Release;Platform=x64
+cmake -BBuild/Release/$ARCH/ -Hsrc/OSPSuite.SimModelNative/ -DCMAKE_BUILD_TYPE=Release -DFuncParserDir=Build/Release/$ARCH/ -DEXT=$EXT
+cp packages/OSPSuite.FuncParser.$FUNCPARSER_DIR/OSPSuite.FuncParserNative/bin/native/$ARCH/Release/libOSPSuite.FuncParserNative.$EXT Build/Release/$ARCH/
+cp packages/OSPSuite.SimModelSolver_CVODES.$CVODES_DIR/OSPSuite.SimModelSolver_CVODES/bin/native/$ARCH/Release/libOSPSuite.SimModelSolver_CVODES.$EXT Build/Release/$ARCH/
+make -C Build/Release/$ARCH/
+dotnet build OSPSuite.SimModel4Nix.sln /property:Configuration=Release
 
-cmake -BBuild/Debug/x64/ -Hsrc/OSPSuite.SimModelNative/ -DCMAKE_BUILD_TYPE=Debug -DFuncParserDir=Build/Debug/x64/ -DEXT=$EXT  -DCMAKE_OSX_ARCHITECTURES=x86_64
-cp packages/OSPSuite.FuncParser.$1/OSPSuite.FuncParserNative/bin/native/x64/Debug/libOSPSuite.FuncParserNative.$EXT Build/Debug/x64/
-cp packages/OSPSuite.SimModelSolver_CVODES.$1/OSPSuite.SimModelSolver_CVODES/bin/native/x64/Debug/libOSPSuite.SimModelSolver_CVODES.$EXT Build/Debug/x64/
-make -C Build/Debug/x64/
-dotnet build OSPSuite.SimModel4Nix.sln /property:Configuration=Debug;Platform=x64
+cmake -BBuild/Debug/$ARCH/ -Hsrc/OSPSuite.SimModelNative/ -DCMAKE_BUILD_TYPE=Debug -DFuncParserDir=Build/Debug/$ARCH/ -DEXT=$EXT
+cp packages/OSPSuite.FuncParser.$FUNCPARSER_DIR/OSPSuite.FuncParserNative/bin/native/$ARCH/Debug/libOSPSuite.FuncParserNative.$EXT Build/Debug/$ARCH/
+cp packages/OSPSuite.SimModelSolver_CVODES.$CVODES_DIR/OSPSuite.SimModelSolver_CVODES/bin/native/$ARCH/Debug/libOSPSuite.SimModelSolver_CVODES.$EXT Build/Debug/$ARCH/
+make -C Build/Debug/$ARCH/
+dotnet build OSPSuite.SimModel4Nix.sln /property:Configuration=Debug
```

Comment:
can be simplified via `for BuildType in Debug Release `
compare https://github.com/Open-Systems-Pharmacology/OSPSuite.FuncParser/blob/d4d86884361d911bebd5b74107a9ada0eda60c17/buildNix.sh#L22C1-L27

Reply:
done